crowbar_framework/app/models/role_object.rb

```diff
@@ -28,24 +28,38 @@ def cluster_roles(roles = RoleObject.all)
     end
   end
 
+  def cluster_remote_roles(roles = RoleObject.all)
+    @cluster_remote_roles ||= begin
```

Comment:
Please, bare with my rusty Ruby, but is this `begin` superfluous here, where there is only one expression?